app/routes/calendar/$id/index.tsx

```diff
@@ -344,3 +350,28 @@ function Description() {
     </Section>
   );
 }
+
+// Check if any team has more than 1 player
+function checkIfIsTeam(
+  results: {
+    teamName: string;
+    placement: number;
+    players: (
+      | string
+      | {
+          id: number;
+          discordName: string;
+          discordDiscriminator: string;
+          discordId: string;
+          discordAvatar: string | null;
+        }
+    )[];
+  }[]
```

Comment:
This is also not a big deal but a small TypeScript tip. I believe you can do something like:

```suggestion
  results: {
    players: unknown[];
  }[]
```

It will same you some typing and makes the function more flexible in some use-cases. TypeScript allows you to pass "more" to the function than you define so it helps to be as unspecific as possible when defining function signature.

Reply:
Thank you so much! I truly appreciate your helpful tips. They are very valuable for a newcomer like me :)